policyengine_us/parameters/gov/states/nd/tax/income/taxable_income/additions/sources.yaml

```diff
@@ -0,0 +1,17 @@
+description: ND additions to US taxable income.
+values:
+  2021-01-01:
+    - c05700  # lump-sum distributions from Federal Form 4972
+
+metadata:
+  unit: variable
+  label: ND additions to US taxable income
+  reference:
+  - title: 2021 ND income tax form
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/form-nd-1-2021.pdf
+  - title: 2021 ND income tax instructions
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/2021-individual-income-tax-booklet.pdf
```

Comment:
If not too much trouble could you add the page number? `#page=x`

Reply:
It is more than a little trouble, but it is done.